### DISCUSSION TITLE
视频合集希望能增加按投稿时间排序功能

### DISCUSSION THREAD

**Discussion by Misaka-Mikoto-Tech** [the1812/Bilibili-Evolved#4063] [功能建议 / Ideas]
现在很多合集的排序是乱的，特别是一些视频教程合集，没有按投稿时间排序，也没有给视频标识序号，导致观看体验痛苦无比，如果能在 
![image](https://user-images.githubusercontent.com/1246379/224556821-e75b05a6-2206-4bb8-967b-e116842cd9e0.png)
这个地方增加一个按投稿时间排序的按钮就会好用很多

**Comment by Geeyun-JY3** [comment]
#2368、https://github.com/the1812/Bilibili-Evolved/discussions/1930#discussioncomment-849494
